src/main/groovy/life/qbic/portal/sampletracking/app/SampleTrackingPortletController.groovy

```diff
@@ -0,0 +1,53 @@
+package life.qbic.portal.sampletracking.app
+
+import life.qbic.datamodel.samples.Location
+import life.qbic.datamodel.samples.Sample
+import life.qbic.datamodel.samples.Status
+import life.qbic.portal.sampletracking.app.samples.list.ModifySampleListInput
+import life.qbic.portal.sampletracking.app.samples.query.SampleTrackingQueryInput
+import life.qbic.portal.sampletracking.app.samples.update.SampleTrackingUpdateInput
+
+class SampleTrackingPortletController implements PortletController {
+
+    private final SampleTrackingUpdateInput sampleUpdateInput
+
+    private final SampleTrackingQueryInput sampleLocation
+
+    private final ModifySampleListInput sampleListModification
+
+    private SampleTrackingPortletController() {
+        new AssertionError()
+    }
+
+    SampleTrackingPortletController(SampleTrackingUpdateInput sampleUpdateInput, SampleTrackingQueryInput sampleLocation, ModifySampleListInput sampleListModification) {
+        this.sampleUpdateInput = sampleUpdateInput
+        this.sampleLocation = sampleLocation
+        this.sampleListModification = sampleListModification
```

Comment:
Sounds good. But I don't know how this will behave when the service cannot be reached. Probably fails and prevents the display of an empty UI (which is to be desired in the finished application but at the moment hinders development since we cannot connect to the server and therefore not see the UI we are building)
Therefore I am not committing this.